### PR TITLE
Feature(permission): permission config team switch

### PR DESCRIPTION
**Pull request by 15168440402**

### CODE REVIEW

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/AddThirdPartyIdpDialog.razor.cs [hunk 1]
```diff
@@ -45,11 +45,11 @@ protected override void OnParametersSet()
 
     public async Task AddThirdPartyIdpAsync(EditContext context)
     {
-        if (string.IsNullOrEmpty(ThirdPartyIdp.Icon))
-        {
-            OpenErrorMessage(T("Please upload ThirdPartyIdp icon"));
-            return;
-        }
+        //if (string.IsNullOrEmpty(ThirdPartyIdp.Icon))
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/AddThirdPartyIdpDialog.razor.cs:48]: Why commit it?

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/UpdateThirdPartyIdpDialog.razor.cs [hunk 2]
```diff
@@ -53,17 +53,17 @@ public async Task GetThirdPartyIdpDetailAsync()
 
     public async Task UpdateThirdPartyIdpAsync(EditContext context)
     {
-        if (string.IsNullOrEmpty(ThirdPartyIdp.Icon))
-        {
-            OpenErrorMessage(T("Please upload ThirdPartyIdp icon"));
-            return;
-        }
+        //if (string.IsNullOrEmpty(ThirdPartyIdp.Icon))
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/UpdateThirdPartyIdpDialog.razor.cs:56]: Why commit it?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/AddThirdPartyIdpDialog.razor.cs (hunk 1 above)
Comment (15168440402) [reply src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/AddThirdPartyIdpDialog.razor.cs:48]: Mr. Xie said the third-party platform icon upload is not required

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/UpdateThirdPartyIdpDialog.razor.cs (hunk 2 above)
Comment (15168440402) [reply src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/ThirdPartyIdps/UpdateThirdPartyIdpDialog.razor.cs:56]: Mr. Xie said the third-party platform icon upload is not required

**Pull request merged**